tests/functional/spec/interstitial_definition.spec.js

```diff
@@ -0,0 +1,43 @@
+import InterstitialDefinitionPage from "../generated_pages/interstitial_definition/interstitial-definition.page";
+
+describe("Component: Interstitial Definition", () => {
+  describe("Given I launch the interstitial definition questionnaire", () => {
```

Comment:
Same in this file, think the reference should be to contents definition as it is not specific to interstitial.